src/openforms/emails/templates/emails/admin_digest.html

```diff
@@ -154,3 +154,44 @@ <h5>{% trans "Logic rules problems" %}</h5>
         {% endfor %}
     </ul>
 {% endif %}
+
+{% if expired_or_near_expiry_referentielijsten_data %}
+    <h4>{% trans "Referentielijsten (tabels-items expired or near expiry)" %}</h4>
+    <ul>
+        {% for service in expired_or_near_expiry_referentielijsten_data %}
+            {% blocktranslate with service_label=service.service %}
+                <h4>Service: {{ service_label }}</h4>
+            {% endblocktranslate %}
+            
+            <!-- Expiring tabels -->
+            {% if service.tabels %}
+                <h4>{% trans "Tabels" %}</h4>
+                {% for tabel in service.tabels %}
+                    <li>
+                        <p>
+                            {% blocktranslate with tabel_naam=tabel.naam eindatum=tabel.eindatum|date trimmed %}
+                            Tabel '{{ tabel_naam }}' expires (or expired) in {{ eindatum }}.<br>
```

Comment:
```suggestion
                            Table '{{ tabel_naam }}' expires (or expired) in {{ eindatum }}.<br>
```
we'll handle the Dutch translations in another commit.

Additionally, we have the information to check if it has expired or will expire soon, so let's use proper messages for that by combining it with the `timesince` and `timeuntil` template filters - it's much more readable to see "will expire in 4 days" or "has expired 7 days ago" than having to compare a date with the current date.